### DISCUSSION TITLE
Where in the code is the decision to do a buy or sell made.

### DISCUSSION THREAD

**Discussion by baerrs** [whittlem/pycryptobot#720] [General]
I'm looking at the code to find where the actual decision is made for a buy or sell.  
Can someone help me please ?

**Comment by baerrs** [comment]
This was answered in the chat (I don't remember who gave the information, sorry I can't give you credit).

/models/Startegy.py

**Reply by whittlem** [reply]
You are right... it's all in models/Strategy.py
